code/game/objects/effects/decals/cleanable.dm

```diff
@@ -7,15 +7,17 @@
 	var/mergeable_decal = TRUE //when two of these are on a same tile or do we need to merge them into just one?
 	var/beauty = 0
 
-/obj/effect/decal/cleanable/Initialize(mapload, list/datum/disease/diseases)
+/obj/effect/decal/cleanable/Initialize(mapload, list/datum/disease/diseases, _step_x, _step_y)

```

Comment:
Avoid adding initialization arguments when you can, they make for brittle code

Reply:
how would you recommend doing this differently?

Reply:
Editing the step vars after creating it is fine

Reply:
ah so call a proc after its created that sets its step vars

Reply:
Sure that's fine but you can just assign to the step vars wherever you create the thing